src/main/java/uk/gov/companieshouse/overseasentitiesapi/model/updatesubmission/changelist/additions/CorporateManagingOfficerAddition.java

```diff
@@ -0,0 +1,60 @@
+package uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.additions;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.commonmodels.Address;
+import uk.gov.companieshouse.overseasentitiesapi.model.updatesubmission.changelist.commonmodels.CompanyIdentification;
+
+import java.time.LocalDate;
+
+public class CorporateManagingOfficerAddition extends ManagingOfficerAddition {
+    @JsonProperty("name")
+    private String name;
+
+    @JsonProperty("contactName")
+    private String contactName;
+
+    @JsonProperty("contactEmail")
+    private String contactEmail;
+
+    @JsonProperty("identification")
```

Comment:
According to the documentation
```suggestion
    @JsonProperty("companyIdentification")
```